core/lib/network-request.js

```diff
@@ -614,7 +614,10 @@ class NetworkRequest {
    * @return {boolean}
    */
   static isContentEncoded(record) {
-    return record.responseHeaders.some(item => item.name === 'Content-Encoding');
+    // FYI: older devtools logs (like our test fixtures) seems to be lower case, while modern logs
+    // are Cased-Like-This.
+    const pattern = /^content-encoding$/i;
+    return record.responseHeaders.some(item => item.name.match(pattern));
```

Comment:
nit: `item.name.toLowerCase()` has worked in the past. Just easier to read than a regex IMO but up to you